packages/http/src/webauthn.ts

```diff
@@ -0,0 +1,43 @@
+import type { definitions } from "./__generated__/services/coordinator/public/v1/public_api.types";
+import { get } from "@github/webauthn-json/browser-ponyfill";
+import { base64StringToBase64UrlEncodedString } from "./encoding";
+
+type TWebAuthnStamp = definitions["v1WebAuthnStamp"];
+
+const timeout = 5 * 60 * 1000;
+
+async function getCredentialRequestOptions(
+  payload: string
+): Promise<CredentialRequestOptions> {
+  const challenge = await getChallengeFromPayload(payload);
+
+  return {
+    publicKey: {
+      challenge: await new TextEncoder().encode(challenge),
+      timeout,
+      userVerification: "discouraged",
```

Comment:
This probably needs to be configurable. In the case of passkeys I don't think it hurts to have `userVerification` set to "preferred" or even "required". It's only in the case of yubikeys that UV is problematic (users may not want to go through the hassle of an extra pin code to remember)

Reply:
I was planning on adding an options param that allows users to add their own signing options. Do you think "preferred" is a sane default?